### PR TITLE
bombastic-walker : long running mode

### PR DESCRIPTION
**Pull request by jbtrystram**
After spending a while working on adapting ctron's CSAF walker i ended up reverting back and simply write a wrapper around shell scripts, mainly for 2 reasons : 

- less complexity, as built in tools like `gpg`, `sha256sum`, `curl` do the job well and in one line of code
- It's currently based on prodsec publishing of files, which is beta, and I can't use the `metadata.json` to find signature and path to files, so it's a bit hacky. If prodsec changes that later, i did not want to invest too much time.

Signed-off-by: jbtrystram <jbtrystram@redhat.com>

### CODE REVIEW

File: bombastic/walker/walker.sh [hunk 1]
```diff
@@ -18,27 +27,24 @@ help()
...
+   echo "sbom_uploader [-h] SBOM_URL BOMBASTIC_API_URL"
    echo "options:"
-   echo "   -s <URL>  Specify the source URL. Defaults to https://access.redhat.com/security/data/sbom/beta/index.txt"
    echo "   -h        Print this Help."
    echo
    echo "example:"
-   echo "sbomwalker localhost:8080"
-   echo "sbomwalker -s https://access.redhat.com/security/data/sbom/beta/index.txt localhost:8080"
+   echo "sbomwalker https://access.redhat.com/security/data/sbom/beta/sbdx/3amp-2.json.bz2 localhost:8080"
    echo
```
Comment (lulf) [inline bombastic/walker/walker.sh:36]: Which one is it, sbom_uploader or sbomwalker? :)

**Review by lulf — CHANGES_REQUESTED** [review]
I'm still of the opinion that  modifying the csaf-walker to support this use case would be better in the long run. And although having the CLI consistent is an improvement, it's a bit brittle the way that it needs shellscript on $PATH or else it will look in a fixed location. 

That being said, it's better than what is there, but I'd like these changes before merging:

* Allow overriding the path to the shell script as a path argument in case we need that to workaround issues in the runtime environment.
* Make it possible to configure the working directory for where files are downloaded, and the GPG 'folder'. There is an issue with the current walker that prevents it from being run on Kubernetes. I "fixed" this locally by setting WORKDIR in the Containerfile and skipping verification, but it should be supported by the tool.
* Make sure the required packages are part of trust/Containerfile (gpg, wget?) and remove the walker/Containerfile.

File: bombastic/walker/walker.sh (hunk 1 above)
Comment (jbtrystram) [reply bombastic/walker/walker.sh]: Fixed with `$0`  :)

**Review by lulf — APPROVED** [review]
Thanks! :rocket:

**Pull request merged**